### PR TITLE
Testing - First PR for issue #476

### PR DESCRIPTION
**Pull request by faytey**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Bugfix
- Feature
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no API changes)
- Build-related changes
- Documentation content changes
- Testing
- Other (please describe):

## What is the current behavior?

Resolves: #NA

## What is the new behavior?

-
-
-

### CODE REVIEW

**Review by Sk8erboi84 — COMMENTED** [review]
fix the scarb fmt please

**Review by Sk8erboi84 — COMMENTED** [review]
.

**Comment by faytey** [comment]
ok, on it now.

**Comment by faytey** [comment]
done, kindly check and review

File: tests/adl/test_adl_utils.cairo [hunk 1]
```diff
@@ -221,18 +225,66 @@ fn given_non_valid_position_when_create_adl_order_then_fails() {
...
+        data_store: data_store,
+        event_emitter: event_emitter,
+        account: account1,
+        market: market,
+        collateral_token: collateral_token,
+        is_long: true,
+        size_delta_usd: 0,
+        updated_at_block: 100
+    };
+    adl_utils::create_adl_order(params);
```
Comment (zarboq) [inline tests/adl/test_adl_utils.cairo:244]:
```suggestion
    let key = adl_utils::create_adl_order(params);

    // Assertions
    let order = data_store.get_order(key);
    assert(order.account == account1, 'wrong order');
    assert(order.order_type == OrderType::MarketDecrease(()), 'wrong type');
    assert(order.updated_at_block == 100, 'wrong updated');
```

**Review by zarboq — CHANGES_REQUESTED** [review]

File: tests/adl/test_adl_utils.cairo (hunk 1 above)
Comment (faytey) [reply tests/adl/test_adl_utils.cairo:244]: alright, fixing that now, is that all the changes that's needed.

**Review by zarboq — APPROVED** [review]
Resolve conflicts then it will be good !

**Comment by faytey** [comment]
hi @zarboq , i synced the fork and realized it closed the pr, so sorry about that, here's the link to the new pr with all conflicts resolved, thanks

**Comment by faytey** [comment]
https://github.com/keep-starknet-strange/satoru/pull/578

**Pull request closed without merging**